### PR TITLE
Remove wip tag

### PR DESCRIPTION
**Pull request by danascheider**
## Summary

This PR removes the `@wip` tag from `retry_failing_tests.feature`.

## Details

Since `retry_failing_tests.feature` is now fully implemented and passes, it should not be tagged as `@wip` as this will break the build after #981 is merged.

## Motivation and Context

Since we have decided not to allow `@wip` features on master, we need to remove the `@wip` tag from features already on master (and, of course, fix ones that don't pass). This will prevent the build from breaking due to the presence of features that should not have been merged. (Something tells me this whole paragraph doesn't make much sense.)

## How Has This Been Tested?

The feature was already passing, so removing the tag doesn't require additional tests.

### CODE REVIEW

File: History.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 ### Bugfixes
```
Comment (danascheider) [inline History.md]: This change was not supposed to be in this PR! I thought I had pushed this directly to master previously :(

**Comment by danascheider** [comment]
Oh no, looks like this feature doesn't, in fact, pass, which means we probably shouldn't have merged #920. I'll have to look into what's going on.

**Comment by brasmusson** [comment]
@danascheider The retry filter is never added to the filters in the [runtime](https://github.com/cucumber/cucumber-ruby/blob/master/lib/cucumber/runtime.rb#L228-L246). Your step definitions for the retry feature does not work. One problem is that their state is not reset between scenarios. I also made [this comment](https://github.com/cucumber/cucumber-ruby/pull/920/files#r64802812) on the PR.

**Comment by danascheider** [comment]
Oh sorry, I didn't see that! I'll check it out when I get to the office.

**Comment by mattwynne** [comment]
See also #981 :)

**Comment by mattwynne** [comment]
Closed in favour of #981

**Pull request closed without merging**